gravitee-apim-gateway/gravitee-apim-gateway-services/gravitee-apim-gateway-services-debug/src/main/java/io/gravitee/gateway/debug/reactor/processor/DebugEventCompletionProcessor.java

```diff
@@ -106,6 +108,14 @@ private io.gravitee.definition.model.debug.DebugApi computeDebugApiEventPayload(
         return debugApi;
     }
 
+    private Buffer getResponseBuffer(DebugExecutionContext debugContext) {
+        Response response = debugContext.response();
+        if (response instanceof TimeoutServerResponseDebugDecorator) {
+            response = ((TimeoutServerResponseDebugDecorator) response).response();
+        }
+        return ((VertxHttpServerResponseDebugDecorator) response).getBuffer();
+    }
+
```

Comment:
Is it related to the following issue https://github.com/gravitee-io/issues/issues/8185 ?